src/test/java/resources/voucherManagementTest.feature

```diff
@@ -0,0 +1,74 @@
+Feature: Voucher Management Api Test
+
+   Scenario: Crate Voucher Api Test
+    Given I can create an VoucherRequestDTO for voucher creation
+    Then I can inject MockServer
+    And I can start mock server
+    When I call the create voucher API with expected status of 202 and stub "/createVoucher"
+    And I can register the stub with "/createVoucher" endpoint for "PUT" request with status of 200
```

Comment:
here we may have to register the stubs early(as  background) enough to get high success rate